README.md

```diff
@@ -17,6 +17,12 @@ See [prerequisites](docs/index.md#prerequisites).
 - [Create a minimal Docker CLI extension](docs/tutorials/minimal-frontend-using-docker-cli.md) - a minimal Desktop Extension containing only a UI part that invokes Docker CLI commands.
 - [Create a ReactJS-based extension](docs/tutorials/react-extension.md) - a minimal Desktop Extension containing only a UI part based on ReactJS.
 
+## Extensions SDK documentation
+
+Documentation about extension SDK and creating your own extensions can be found [here](https://docs.docker.com/desktop/extensions-sdk/).
+
+Pull Requests are welcome to update/improve documentation content at https://github.com/docker/docker.github.io/tree/master/desktop/extensions-sdk
```

Comment:
Could we change this to keep it inline with the docker.github.io contribution guidelines which asks that updates/improvements are made against the master branch? https://github.com/docker/docker.github.io/blob/master/CONTRIBUTING.md

Reply:
Ah, good, I can point to this CONTRIBUTING page indeed